templates/shibboleth/conf/authn/general-authn.xml.erb

```diff
@@ -55,14 +55,16 @@
             p:passiveAuthenticationSupported="true"
             p:forcedAuthenticationSupported="true"
             p:nonBrowserSupported="false" />
-          <!-- <property name="supportedPrincipals">
+           <%- @nih_refeds_profile -%>
```

Comment:
```suggestion
           <%- if @nih_refeds_profile -%>
```

Reply:
@edestecd Are you good with this now? We'd like to get it merged.

Reply:
Yea it looks to be fixed